game.py

```diff
@@ -20,7 +20,32 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    correct_guesses = 0
+    wrong_guesses_list = []
+    waiting_for_correct_lengths = True
+
+    while waiting_for_correct_lengths and len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES:
+
+        user_input_letter = get_letter_from_user(correct_letter_guess_statuses, wrong_guesses_list)
+        if user_input_letter in correct_letter_guess_statuses:
```

Comment:
For readability I would add a space between lines like these. It helps break up your code into sections and groups things based on their responsibility.
```suggestion
        user_input_letter = get_letter_from_user(correct_letter_guess_statuses, wrong_guesses_list)
        
        if user_input_letter in correct_letter_guess_statuses:
```

Reply:
Added